pyam/plotting.py

```diff
@@ -437,7 +437,7 @@ def stack(
 
     def as_series(index, name):
         _idx = [i[0] for i in index]
-        return pd.Series([0] * len(index), index=_idx, name=name)
+        return pd.Series([0] * len(index), index=_idx, name=name, dtype="int")
```

Comment:
Not clear why this is needed, can you let me know why?

Can this be sent through calling code? e.g., either with

1. move this `dtype` up as a kwarg or
2. add generic `**kwargs` which then passes through to this series call

Reply:
This is a function only used within this function (see just the following line) to select the rows where a timeseries crosses zero. Adding the dtype explicitly is done just to silence a warning in the log.

Reply:
Gotcha - won't hold up. But I assume this will not work for non-integer times?

Reply:
*facepalm* reread, and realize no issue here. sorry!